tpl/widget/header/categorylist.tpl

```diff
@@ -31,7 +31,7 @@
                                 [{foreach from=$oxcmp_categories item="ocat" key="catkey" name="root"}]
                                     [{if $ocat->getIsVisible()}]
                                         [{foreach from=$ocat->getContentCats() item="oTopCont" name="MoreTopCms"}]
-                                            <li>
+                                            <li [{if $oContent->oxcontents__oxloadid->value===$oTopCont->oxcontents__oxloadid->value}]class="active"[{/if}]>
```

Comment:
I would change the formatting to: `<li[{if $oContent->oxcontents__oxloadid->value === $oTopCont->oxcontents__oxloadid->value}] class="active"[{/if}]>`

Reply:
ok